# This service returns the best estimate for where an object is in a scene 
#  using multiple view points. Each view is stored in a point cloud file.
#  The first file is the reference. It is assumed that the current bounds
#  are what will be used for all of the files (transformed, of course).
#  This service will transform each point cloud file into the reference
#  frame of the first point cloud file. It will then use the aggregated
#  points to find the best fit for an object, and return the pose of 
#  that object in the world frame

# This service takes in a list of all of the file names containing the
#  views, and the transform of where each view was taken

# Note that it is also assumed that the bounding box works 
#  for the first taken frame

int64 num_views         # The number of views to process
string[] filenames      # List of point cloud files
float64[] trans         # Array of length 3*N, stores the translation from the world frame that each set of points was captured at
float64[] quat          # Array of length 4*N, stores the rotation from the world frame that each set of points was captured at
string cameraName       # The camera that the points were captured from

----

int64 objNum            # What object we see
float64[3] trans        # The translation of the object in the world frame
float64[4] quat         # Rotation of the object in the world frame

int64 ret
string msg
